doc/hld_template.md

```diff
@@ -0,0 +1,65 @@
+# HLD Name #
+
+## Table of Content 
+
+### 1. Revision  
+
+### 2. Scope  
+
+This section describes the scope of this high-level design document in SONiC.
+
+### 3. Definitions/Abbreviations 
+
+This section covers the abbreviation if any, used in this high-level design document and its definitions.
+
+### 1. Overview 
+
+The purpose of this section is to give an overview of high-level design document and its architecture implementation in SONiC. 
+
+### 2. Requirements
+
+This section list out the basic requirements for the HLD coverage and exemptions (not supported) if any for this design.
```

Comment:
Suggest you remove the word "basic" - we want *all* the requirements.

Reply:
Done.

Reply:
Ack